socless/integrations.py

```diff
@@ -219,22 +221,38 @@ def __init__(self, event, lambda_context, integration_handler, include_event=Fal
             self.event = event
 
         self.testing = bool(self.event.get("_testing"))
+        self.execution_id = self.event.get("execution_id", "")
+
         try:
             self.state_config = self.event["State_Config"]
-        except:
-            raise KeyError("No State_Config was passed to the integration")
+        except KeyError:
+            # not triggered from socless playbook (direct invoke via CLI, Test console, etc.)
+            if "execution_id" not in self.event and "artifacts" not in self.event:
+                print(
```

Comment:
Let's use socless_log.info here please instead of print please. Forgot to add that feedback on my first pass 🤦‍♂️